promql/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM golang:1.16 as build-env
+
+WORKDIR /go/src/promql
+COPY . /go/src/promql
+
```

Had to disable CGO to make this work because the container used later on probably doesn't have a "proper" libc:
```suggestion
ENV CGO_ENABLED 0
```